umbraco/Umbraco.CMS.Backoffice
# Bugfix: RTE TinyMCE data-type sorts configuration field order

## Description

In v14, the order of the RTE TinyMCE configuration fields was skewed due to the combination of the `Umbraco.RichText` schema properties. This PR adds a weighting to each of the properties to correct the order so that it matches the order in Umbraco v13 (and prior).

## Types of changes


## How to test?

Go to a TinyMCE RTE data-type, notice that the order should be...

1. Toolbar
2. Stylesheets
3. Dimensions
4. Maximum size for inserted images
5. Mode
6. Available Blocks
7. Overlay Size
8. Hide Label
9. Image Upload Folder
10. Ignore User Start Nodes
